packit/utils/repo.py

```diff
@@ -29,12 +29,14 @@ class RepositoryCache:
     * Project name is used to match the git project in the cache.
     """
 
-    def __init__(self, cache_path: Path, add_new=False) -> None:
-        self.cache_path = cache_path
+    def __init__(self, cache_path: Union[str, Path], add_new=False) -> None:
+        self.cache_path = (
+            Path(cache_path) if isinstance(cache_path, str) else cache_path
+        )
```

Comment:
```suggestion
        self.cache_path = Path(cache_path)
```

```python
In [1]: from pathlib import Path                                                                                                                                                                                                              

In [2]: p=Path('.')                                                                                                                                                                                                                           

In [3]: p                                                                                                                                                                                                                                     
Out[3]: PosixPath('.')

In [4]: Path(p)                                                                                                                                                                                                                               
Out[4]: PosixPath('.')
```

Reply:
hehe, neat

Reply:
I had this before but it did not work with `flexmock` in unit tests...